tests/models/test_grant.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""Unit tests for Grant model."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from ..factories import GrantFactory
+
+
+@pytest.mark.usefixtures('db', 'user')
+def test_get_by_id(user):
```

Comment:
user is not needed

Reply:
Or... `token = Token(user_id=user.id, client=client, scopes='')`

Reply:
Fixed.